test/http-api/extra/block.js

```diff
@@ -8,8 +8,13 @@ chai.use(dirtyChai)
 
 const multihash = require('multihashes')
 const waterfall = require('async/waterfall')
+const getCtl = require('./utils/get-ctl.js')
 
-module.exports = (ctl) => {
+module.exports = (http) => {
+  let ctl = null
+  before(() => {
+    ctl = getCtl(http)
+  })
```

Comment:
This can be simplified if https://github.com/ipfs/js-ipfs/pull/1103/files#r169997744 gets done.

Reply:
Not sure how that would work as we would still need the address to connect to js-ipfsd-ctl from the tests?

Reply:
ipfsd-ctl exposes the client automatically on ipfsd.api. It can work in the same way as before, by just passing the client.